python/ml4ir/applications/ranking/model/metrics/metrics_helper.py

```diff
@@ -224,13 +233,13 @@ def compute_secondary_labels_metrics_on_query_group(
 
 
 def get_grouped_stats(
-    df: pd.DataFrame,
-    query_key_col: str,
-    label_col: str,
-    old_rank_col: str,
-    new_rank_col: str,
-    group_keys: List[str] = [],
-    secondary_labels: List[str] = [],
+        df: pd.DataFrame,
```

Comment:
I notice a lot of indentations added. Is it auto formatting?

Reply:
Yes, auto formatting indeed